RetailSuiteAccounting/Integration/07_ManageBWAReports.md

```diff
@@ -0,0 +1,139 @@
+[!!User Interface BWA assignment](../UserInterface/02i_BWAAssignment.md)  
+[!!User Interface evaluations](../UserInterface/01_Book.md#evaluations)  
+[!!Create a PDF evaluation](../Operation/15_ProvideAccountingData.md#create-a-pdf-evaluation)  
+
+
+# Manage the BWA reports
+
+[comment]: <> (BWA reports or Accounting Reports? See Accounting Reports view -> ZUORDNUNG BWA tab)
+
+The business evaluation (BWA) is a reporting system based on company financial data and provides information about the company's current cost and revenue situation and, consequently, its  profitability. BWA reports serve as a basis for managerial future decisions, and they are widely used by capital investors and credit institutions for company assessment. BWA is an essential planning and controlling tool for the company's future development.
+
+The *BWA assignment* tab in the *Accounting* module enables to create new BWA reports, to copy one of the predefined system reports and edit it, or to delete any of them. All available reports can be completely customized.
+
+ > [Warning] Be aware that any deletion is permanent and cannot be undone, and any changes made will overwrite the preset values.
+
+When customizing existing reports, bear in mind that the reports use the standard accounts set up in the *Accounting* module configuration process. For detailed information, see [Charts of accounts](./01_RunAccountingWizard.md#chart-of-accounts). If the accounts have been customized manually, they will have to be set up again individually for the BWA reports. Otherwise, the BWA reports will not recognized them.
```

Comment:
```suggestion
When customizing existing reports, bear in mind that the reports use the standard accounts set up in the *Accounting* module configuration process. For detailed information, see [Chart of accounts](./01_RunAccountingWizard.md#chart-of-accounts). If the accounts have been customized manually, they will have to be set up again individually for the BWA reports. Otherwise, the BWA reports will not recognized them.
```